src/comm.cpp

```diff
@@ -605,6 +608,24 @@ void game_loop(int mother_desc)
       /* figure out for how long we have to sleep */
       gettimeofday(&now, (struct timezone *) 0);
       timespent = timediff(&now, &last_time);
+
+      {
+        long int total_usec = 1000000 * timespent.tv_sec + timespent.tv_usec;
+        double usage_pcnt = 100 * ((double)total_usec / OPT_USEC);
+        PERF_log_pulse(usage_pcnt);
+
+        if (usage_pcnt >= 100)
+        {
+          log("Pulse usage >= 100%%. Trace info: ");
+          char buf[MAX_STRING_LENGTH * 12];
```

Comment:
M_S_L is currently defined as a rather beefy 32768 characters. This makes the buf declared here a 393kb array; is that required?

Reply:
MSL * 12 was basically the max page string length from Aarchon. Will just use MSL here.